sdk/keyvault/azure-security-keyvault-certificates/README.md

```diff
@@ -1,5 +1,5 @@
 # Azure Key Vault Certificate client library for Java
-Azure Key Vault allows you to create and store certificates in the Key Vault. Azure Key Vault client supports certificates backed by Rsa keys and Ec keys. It allows you to securely manage, tightly control your certificates.
+Azure Key Vault allows you to create and store certificates in the Key Vault. Azure Key Vault Certificate client Library supports certificates backed by Rsa keys and Ec keys. It allows you to securely manage, tightly control your certificates.
```

Comment:
Here's how I would rewrite this: 

Azure Key Vault allows you to securely manage and tightly control your certificates.  Azure Key Vault Certificate client library supports certificates backed by RSA and EC keys.